rebar.config

```
@@ -1,4 +1,4 @@
 {erl_opts, [debug_info]}.
 {deps, [
-       {jesse, {git, "https://github.com/for-GET/jesse.git", {branch, "master"}}}
-       ]}.
\ No newline at end of file
+       {jesse, {git, "https://github.com/for-GET/jesse.git", {tag, "1.8.1"}}}
```

Takes it from hex instead

```suggestion
       {jesse, "1.8.1"}
```